backend/overstroomik_service/main.py

```diff
@@ -34,12 +35,19 @@ async def read_root():
 @app.get('/by_rd', response_model=FloodInfo)
 async def by_rd(x: Optional[float] = None, y: Optional[float] = None) -> FloodInfo:
 
-    # Get location information from the geoserver
-    data = Data()
+    # determine which pdok function by input
+    if x and y:   
```

Comment:
Does the `Optional[float]` for `x` and `y` come from the OpenAPI spec? I'd say make these required and then we also don't need this check.